opentmi_client/main.py

```diff
@@ -4,86 +4,185 @@
 MIT
 """
 
+from __future__ import print_function
+import os
 import sys
 import json
 import argparse
+import logging
+import logging.handlers
+from urlparse import urlparse
```

Comment:
Why are we importing both logging and logging.handlers? Unused import at urlparse?